blog/secadvisor.md

```diff
@@ -0,0 +1,290 @@
+
+Be sure to set the parameters max_flow_array_size, etc, to reasonable values.
+
+The max_flow_array_size specifies the maximum number of flows that will be stored in each iteration.
+If max_flow_array_size is set to 0 (or not set at all), then the number of flows that can be stored is 0, and hence no useful information will be saved in the object store.
+This will result in an error.
+
+The max_flows_per_object parameter specifies the number of flows that may stored in a single object. If there are more flows found in a single iteration, then several objects will be created, each with up to max_flows_per_object flows in them. In order to have all the flows stored in a single object, be sure that this parameter is set sufficiently large.
+If max_flows_per_object is set to 0 (or not set at all), then each object will be able to hold 0 flows; i.e. no flows will be able to be stored in any object.
+This will result in an error.
+
+The flow information is collected in groups of objects (called streams) determined by the max_seconds_per_stream parameter.
+All the objects generated within the number of seconds specified in the max_seconds_per_stream parameter are collected under the same heading (stream) in the object url path.
+If max_seconds_per_stream is set to a very large number, then all of the flows captured in the current run of the Security Advisor will be included in the same collection (stream)
+Furthermore, a single object may contain multiple statistics of the flows - all those collected within max_seconds_per_object seconds. Thus if max_seconds_per_object is set to 200 and the flows statistics  are captured once a minute, then each object will contain 3 or 4 sets of statistics - whatever was collected within 200 seconds. 
+
+## Setup Object Store
+
+The Security Advisor saves data to an S3-type object store.
+The parameters to access the object store must be provided in the secadvisor.yml configuration file.
+In case the user does not already have an object store, we show below how to create an object store for testing purposes.
+
+You can either use AWS-style HMAC keys by setting "access_key" and "secret_key", or use IBM IAM OAuth by setting "api_key".
+
+### Setup IBM COS
+
+In IBM Cloud, Create an Object Store resource. The Lite plan gives limited resources for free.
+In the Object Store, create a bucket to hold the Skydive flow information. For simple testing, you can choose Single Site resiliency.
+Look under Bucket Configuration to see the Public endpoint (url) where the bucket is accessed. This endpoint information needs to go into the secadvisor.yml "endpoint" field.
+Go to the "Service Credentials" panel and create a new credential. Click on "View Credential" to get the details of the credential. The apikey needs to go into the secadvisor.yml "api_key" field.
+In the secadvisor.yml file, uncomment the iam_endpoint field and set it to https://iam.cloud.ibm.com/identity/token.
+
+### Setup Minio
```

Comment:
the content here is incorrect - please see the content of my README

Reply:
OK. Added a pointer to the README and deleted the rest of this section.